app/Http/Controllers/Api/V1/SocialLoginController.php

```diff
@@ -131,12 +131,14 @@ public function dtaLogin(Request $request, string $provider): mixed
      */
     public function login(Request $request, string $provider): mixed
     {
-        return $this->handleLogin($request, $provider, env('GATEWAY_URL'), env('OPENATHENS_REDIRECT_URL'));
+        return $this->handleLogin($request, $provider, env('GATEWAY_URL'), env('OPENATHENS_REDIRECT_URL'), false);
 
     }
 
-    private function handleLogin(Request $request, string $provider, string $baseRedirectUrl, $openAthensRedirectUrl): mixed
+    private function handleLogin(Request $request, string $provider, string $baseRedirectUrl, $openAthensRedirectUrl, $isDTA): mixed
```

Comment:
Future reference, you could have provided a default value for `$isDTA` and thus not need to update the original call signature.